README.md

```diff
@@ -1,2 +1,109 @@
-# KokkosP Profiling Tools
-Performance profiling tools that can be optionally integrated into Kokkos-based applications and libraries.
+
+
+# Kokkos Tools
+
+Kokkos Tools provide a light-weight set of profiling and debugging utilities, which interface with instrumentation hooks built directly into the Kokkos runtime. Compared to 3rd party tools these tools can provide much cleaner, context-specific information: in particular, they allow kernel-centric analysis and they use labels provided to Kokkos objects (kernel launches and views).
+
+Under most circumstances, the profiling hooks are compiled into Kokkos executables by default assuming that the profiling hooks' version is compatible with the tools' version. No recompilation or changes to your build procedures are required.
+
+Note: `Kokkos` must be configured with `Kokkos_ENABLE_LIBDL=ON` to load profiling hooks dynamically. This is the default for most cases anyway.
+
+## General Usage
+
+ To use one of the tools you have to compile it, which will generate a dynamic library. Before executing the Kokkos application you then have to set the environment variable `KOKKOS_TOOLS_LIB` to point to the dynamic library e.g. in Bash:
+```
+export KOKKOS_PROFILE_LIBRARY=${HOME}/kokkos-tools/src/tools/memory-events/kp_memory_event.so
```

Comment:
I think it should be 
```suggestion
export KOKKOS_TOOLS_LIB=${HOME}/kokkos-tools/src/tools/memory-events/kp_memory_event.so
```

Reply:
Thanks. Yes, it is 'Kokkos_TOOLS_LIB`, which is what I have used for testing with Kokkos Tools on various platforms.

The use of Kokkos_Profile_Library may be a copy-paste error (possibly since some of the README is copy-pasted from the Kokkos Tools wiki).